libs/template/renderer.go

```diff
@@ -153,12 +153,17 @@ func (r *renderer) computeFile(relPathTemplate string) (file, error) {
 		return nil, err
 	}
 
+	rootPath, err := filepath.Abs(r.instanceRoot)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Please include a comment saying why we need the abs path.

A reader may ponder that as long as everything is relative to CWD anyway, why care?